README.md

```diff
@@ -0,0 +1,56 @@
+# eslint-config-feedzai
+
+This package provides the ESLint configurations as an extensible shared configuration that could be used by everyone and across the Feedzai's products.
```

Comment:
I would reword this as: "This repository contains the ESLint configuration used across products at Feedzai. We share this configuration to promote the use of linters in JavaScript and to share our conventions and best practices in the hope of inspiring others when defining their configurations."